core/src/components/checkbox/checkbox.tsx

```diff
@@ -22,6 +23,7 @@ import { createColorClasses, hostContext } from '../../utils/theme';
 @Component({
   tag: 'ion-checkbox',
   styleUrls: {
+    base: 'checkbox.scss',
```

Comment:
I'd say this is intentional. Ionic renders a native checkbox under the hood, but we don't use that as the basis for styling since the iOS/MD styles are so unique. As a result, the checkbox appears hidden when using base components. I'm hesitant to add a "default look" for this as it may add too much opinionation to the default styles.

I expect that certain components developers may choose to leave alone. If you look at the Spotify app, they chose to customize everything but the standard form controls:

![IMG_1818](https://user-images.githubusercontent.com/2721089/214392754-53eda371-8139-4446-a989-fd1e9a6d4a68.PNG)

Thoughts on leaving it as-is for now and then letting devs give us feedback?

Reply:
Works for me 👍 As long as we're intentionally taking that stance.

Reply:
I'll call this out in the base component docs. I could see a docs improvement where we highlight the architecture of the component so complex components are a bit easier to customize